ui/app/components/app/permission-page-container/index.scss

```diff
@@ -173,23 +173,32 @@
       }
 
       &__identicon-container, .icon-with-fallback__identicon-container {
-          padding: 1rem;
-          flex: 1;
-          position: relative;
-          width: 100%;
-          display: flex;
-          justify-content: center;
-          align-items: center;
+        padding: 1rem;
+        flex: 1;
+        position: relative;
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
       }
 
       &__identicon-border, .icon-with-fallback__identicon-border {
-          height: 64px;
-          width: 64px;
-          border-radius: 50%;
-          border: 1px solid white;
-          position: absolute;
-          background: #FFFFFF;
-          box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
+        height: 64px;
+        width: 64px;
+        border-radius: 50%;
+        border: 1px solid white;
+        background: #FFFFFF;
+        box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
+      }
+
+      &__identicon-border {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .icon-with-fallback__identicon-border {
+        position: absolute;
```

Comment:
I left alone the `icon-with-fallback` styles. I think it'd be nicer to update all of our icon borders to work this way as well, but that can come in a later PR.